include/wasm.h

```diff
@@ -301,8 +301,6 @@ typedef struct wasm_val_t {
   union {
     int32_t i32;
     int64_t i64;
-    float32_t f32;
```

Comment:
I'm not sure why it's necessary to remove the float fields. We can provide the bits accessors without requiring accessors for the common case, which is less convenient and difficult for FFI bindings, which cannot access inline functions.